### ISSUE TITLE
Website bugs on mobile 

### ISSUE THREAD

**Issue by Gaelan** [DSACMS/open#23]
Tried to show someone the site on my phone, and noticed a couple bugs:
* the “menu” button doesn’t work
* the project cards are too wide, causing a horizontal scroll bar

**Comment by Gaelan** [comment]
Project cards being too wide: fixed by https://github.com/FullHuman/purgecss/pull/1133

**Comment by natalialuzuriaga** [comment]
7/15:
- Starting on this today
- We can apply bug fixes for this to metrics site

**Comment by Firebird1029** [comment]
https://github.com/DSACMS/open/pull/35 is ready for review, I am not sure how to deploy to prod but once done the issues should be fixed

**Comment by natalialuzuriaga** [comment]
7/19:
- PR is ready for review

**Comment by decause-gov** [comment]
7/22:
- @natalialuzuriaga will review
- Once merged, we can close!

**Comment by Firebird1029** [comment]
@Firebird1029 take before and after pics of website

**Comment by decause-gov** [comment]
07/26:
- Re-adding the easi-app repo that was moved into repos.js
- For now, replacing the repo with the location it's moved to, but, there is an API endpoint to resolve this. Discussion in https://github.com/DSACMS/open/pull/35#pullrequestreview-2199403627
- ACTION: file a new ticket as per discussion link above, edit the repos.js file, and then we close this one after merge 🐛

**Comment by Firebird1029** [comment]
Created new ticket, PR is ready for re-review: https://github.com/DSACMS/open/pull/35

**Comment by decause-gov** [comment]
07/29:
- New issue filed, PR sent. Changed repo name.
- Blocking on PR review (@natalialuzuriaga will review)
- Goal: Push to main, update live site.

**Comment by decause-gov** [comment]
08/02:
- PR is merged! Won't show on live site until there are config changes.
- The metrics site, when Gaelan built the Open website, there were specific tools, but there are bugs now. CSS bugs, pass-through bugs, etc, that were made with Metrics, but not with /Open site. Nat is working on them!
- Leaving this open til the build tools updates are complete.
- IDEA: use the same build tools as metrics.

**Comment by decause-gov** [comment]
08/03:
- Waiting on Nat to return 8/12

**Comment by IsaacMilarky** [comment]
8/9:

- Build hasn't worked but PR has been merged
- Still pending

**Comment by natalialuzuriaga** [comment]
8/12:
- Meeting on the books later today
